junebug/logging_service.py

```diff
@@ -50,7 +50,7 @@ def logger_for_event(self, event):
         system = event.get('system', '-')
         parts = [self.worker_id]
         if system != '-':
-            parts.extend(system.split(','))
+            parts = system.split(',')
```

Comment:
Do we still need the `if system != -'` check then? And in what situations can system be `-`? Will all channel workers log these messages where `system` is `-`?